telegram: panic

EditDisable=true

```
[51217] DEBUG telegram:     == Receiving event: (*tgbotapi.Message)(nil)                                             
panic: runtime error: invalid memory address or nil pointer dereference                                              
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xd69b97]                                               

goroutine 130 [running]:      
github.com/42wim/matterbridge/bridge/telegram.(*Btelegram).handleRecv(0xc000511420, 0xc000520360)                    
        /home/matterbridge/.go/src/github.com/42wim/matterbridge/bridge/telegram/handlers.go:129 +0x257              
created by github.com/42wim/matterbridge/bridge/telegram.(*Btelegram).Connect                                        
        /home/matterbridge/.go/src/github.com/42wim/matterbridge/bridge/telegram/telegram.go:46 +0x308  
```

--- a/bridge/telegram/handlers.go
+++ b/bridge/telegram/handlers.go
@@ -109,8 +109,8 @@ func (b *Btelegram) handleRecv(updates <-chan tgbotapi.Update) {
 	for update := range updates {
 		b.Log.Debugf("== Receiving event: %#v", update.Message)
 
-		if update.Message == nil && update.ChannelPost == nil &&
-			update.EditedMessage == nil && update.EditedChannelPost == nil {
+		if update.Message == nil || update.ChannelPost == nil ||
+			update.EditedMessage == nil || update.EditedChannelPost == nil {
 			b.Log.Error("Getting nil messages, this shouldn't happen.")
 			continue
 		}

